controllers/datadogagent/feature/orchestratorexplorer/feature.go

```diff
@@ -71,12 +75,21 @@ func (f *orchestratorExplorerFeature) ID() feature.IDType {
 func (f *orchestratorExplorerFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.RequiredComponents) {
 	f.owner = dda
 	orchestratorExplorer := dda.Spec.Features.OrchestratorExplorer
+	nodeAgent, ok := dda.Spec.Override[v2alpha1.NodeAgentComponentName]
+	// Process Agent is not required on versions as of 7.51.0
+	f.processAgentRequired = ok && nodeAgent.Image != nil && !utils.IsAboveMinVersion(component.GetAgentVersionFromImage(*nodeAgent.Image), "7.51.0")
```

Comment:
Any risk of `*nodeAgent.Image` being nil?

Reply:
Hopefully someone from container-ecosystems can confirm, but if nodeAgent.Image is not set then `gcr.io/datadoghq/agent:latest` is being used for the image by default. Thus, I have that if `*nodeAgent.Image` is nil, `processAgentRequired` evaluates to false (latest is 7.51.0 as of now).

Reply:
for readability, do you mind structuring the conditional similar to how it's done [here](https://github.com/DataDog/datadog-operator/blob/678c4ecd3356cd0601a7ab837e81a361c6eb351d/controllers/datadogagent/feature/kubernetesstatecore/feature.go#L105-L106)?